vendor/bat-native-ads/src/bat/ads/internal/experiments/experiments_util.h

```diff
@@ -0,0 +1,20 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BAT_ADS_INTERNAL_EXPERIMENTS_EXPERIMENTS_UTIL_H_
+#define BAT_ADS_INTERNAL_EXPERIMENTS_EXPERIMENTS_UTIL_H_
+
+#include <string>
+
+namespace ads {
+namespace experiments {
+
+bool IsActive(
```

Comment:
I think we can just add it to `class Experiment`. 
add a static function which will do exactly the same, and a virtual method that will call a static function and pass the name

Reply:
This is somewhat future-proofed and everything in experiments/* is set up to align with the structure that a refactor by @tmancey will introduce very soon. So experiments should have the same structure as targeting/conversions/frequency caps/etc.